BASE/src/ServerTelemetryChannel/Implementation/TransmissionPolicy/AuthenticationTransmissionPolicy.cs

```diff
@@ -36,49 +36,34 @@ public void Dispose()
 
         private void HandleTransmissionSentEvent(object sender, TransmissionProcessedEventArgs e)
         {
-            if (this.Enabled)
+            if (this.Enabled && e.Response != null)
             {
-                HttpWebResponseWrapper httpWebResponseWrapper = e.Response;
-                if (httpWebResponseWrapper != null)
+                switch (e.Response.StatusCode) // HttpWebResponseWrapper
                 {
-                    if (httpWebResponseWrapper.StatusCode == ResponseStatusCodes.BadRequest)
-                    {
-                        // HTTP/1.1 400 Incorrect API was used - v2 API does not support authentication.
+                    case ResponseStatusCodes.BadRequest:
+                        // "HTTP/1.1 400 Incorrect API was used - v2 API does not support authentication".
                         // This indicates that the AI resource was configured for AAD, but SDK was not.
-                        // This is a configuration issue and is not recoverable. 
-                        this.ApplyHaltPolicy(e);
-                    }
-                    else if (httpWebResponseWrapper.StatusCode == ResponseStatusCodes.Unauthorized)
-                    {
-                        // HTTP/1.1 401 Unauthorized - please provide the valid authorization token
+                        // This is a configuration issue and is not recoverable from the client side. 
+                        // If the customer chooses to disable AAD in the AI Resource, the SDK could resume sending.
+                        this.ApplyThrottlePolicy(e);
+                        break;
+                    case ResponseStatusCodes.Unauthorized:
```

Comment:
shall we write using below syntax, it makes code easier to read.
```
case ResponseStatusCodes.Unauthorized:
case ResponseStatusCodes.Forbidden:
       this.ApplyThrottlePolicy(e);
       break;
```
Ref: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/switch#the-switch-section, like case 2 and 3 combined from doc sample.